src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts

```diff
@@ -1,41 +1,41 @@
+import { Location } from '@angular/common';
 import { HttpClient, HttpErrorResponse, HttpHeaders, HttpResponse } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { GitHubDetails, GitProviderService, HelperService } from 'ngx-launcher';
 import { AuthenticationService } from 'ngx-login-client';
-import { empty as observableEmpty, Observable, of as observableOf,  throwError as observableThrowError } from 'rxjs';
+import { empty as observableEmpty, Observable, of, of as observableOf, throwError as observableThrowError } from 'rxjs';
 import { catchError, map, mergeMap } from 'rxjs/operators';
 
 import { ProviderService } from '../../../shared/account/provider.service';
 
 @Injectable()
 export class AppLauncherGitproviderService implements GitProviderService {
 
-    private END_POINT: string = '';
-    private API_BASE: string = 'services/git/';
-    private ORIGIN: string = '';
-    private PROVIDER: string = 'GitHub';
-    private gitHubUserLogin: string;
-    private headers: HttpHeaders = new HttpHeaders({
-      'Content-Type': 'application/json',
-      'X-App': 'osio',
-      'x-git-provider': this.PROVIDER
-    });
-
-
-    constructor(
-      private http: HttpClient,
-      private auth: AuthenticationService,
-      private helperService: HelperService,
-      private providerService: ProviderService
-    ) {
-      if (this.helperService) {
-        this.END_POINT = this.helperService.getBackendUrl();
-        this.ORIGIN = this.helperService.getOrigin();
-      }
-      if (this.auth.getToken() != null) {
-        this.headers = this.headers.set('Authorization', `Bearer ${this.auth.getToken()}`);
-      }
+  private END_POINT: string = '';
+  private API_BASE: string = 'services/git/';
+  private PROVIDER: string = 'GitHub';
+  private headers: HttpHeaders = new HttpHeaders({
+    'Content-Type': 'application/json',
+    'X-App': 'osio',
+    'x-git-provider': this.PROVIDER
+  });
+
+  private gitHubUserLogin: string;
+  private repositories: object = {};
+
+  constructor(
+    private http: HttpClient,
+    private auth: AuthenticationService,
+    private helperService: HelperService,
+    private providerService: ProviderService
+  ) {
+    if (this.helperService) {
+      this.END_POINT = this.helperService.getBackendUrl();
+    }
+    if (this.auth.getToken() != null) {
+      this.headers = this.headers.set('Authorization', `Bearer ${this.auth.getToken()}`);
```

Comment:
We don't need explicit adding of auth headers since we have interceptors for that now.

Reply:
@christianvogt [here](https://github.com/fabric8-ui/fabric8-ui/pull/3393/files#diff-ab395aaaf365a9f0687a1b933c2a925aR16) we are also adding ```x-git-provider``` in header.

Reply:
The git provider header is ok. But we have interceptors that add the author token.

Reply:
Leave as is. We will sweep all auth header additions as we clean up the rest of them later.